soc/xtensa/intel_adsp/tools/cavstool.py

```diff
@@ -339,6 +345,14 @@ def __getattr__(self, name):
 def runx(cmd):
     return subprocess.check_output(cmd, shell=True).decode().rstrip()
 
+def mask(bit):
```

Comment:
Quibble: maybe name this more descriptively than just "mask", it's computing one of the ADSPCS subfields from an input CPU bit.

Reply:
Would `adspcs_mask` improve it? This was another name I thought for this method, but ended up with only `mask` because it could be used somewhere else, but maybe including the register itself. I may be thinking too much in the future, though.

Reply:
Meh.  Not worth a fight.

But it's worth pointing out that when this was a 100-line script it felt perfectly natural to put the ADSPCS bit values inline into code.  Now it's 600+ lines and that seems like it needs to be cleaned up, so what should it look like in a 1000-line script (because you know this will get there)?  Seems like "not mask()" to me. :)